### PR TITLE
Introduce percent class

### PR DESCRIPTION
**Pull request by gzoritchak**

### CODE REVIEW

**Comment by gzoritchak** [comment]
It should be interesting to introduce some coerce function like [kotlin standard lib ones](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-double/index.html).

Another `coerceToDefault` (name to find) to coerce to 0.0 1.0 which is a very standard coercion on percents.

**Comment by gzoritchak** [comment]
Also maybe a `Number.times(Percent)` extension function to be able to do some `2 * percent` call.

**Comment by pmariac** [comment]
DING! 
branch is ready!

**Comment by pmariac** [comment]
@gzoritchak 

please review and merge if OK

File: color/d2v-color-common/src/main/kotlin/io/data2viz/color/HclColor.kt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package io.data2viz.color
 
 import io.data2viz.math.Angle
+import io.data2viz.math.Percent
+import io.data2viz.math.pct
```
Comment (gzoritchak) [inline color/d2v-color-common/src/main/kotlin/io/data2viz/color/HclColor.kt]: We should enforce the good practice of using star imports in data2viz.

**Comment by gzoritchak** [comment]
Splited in few new PRs.

**Pull request closed without merging**